agent/protocol/rocketmq/rocketmq.go

```diff
@@ -0,0 +1,140 @@
+package rocketmq
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"kyanos/agent/buffer"
+	"kyanos/agent/protocol"
+	"kyanos/bpf"
+)
+
+func init() {
+	protocol.ParsersMap[bpf.AgentTrafficProtocolTKProtocolRocketMQ] = func() protocol.ProtocolStreamParser {
+		return &RocketMQStreamParser{}
+	}
+}
+
+func (r *RocketMQMessage) FormatToString() string {
+	return fmt.Sprintf("base=[%s] command=[%s] payload=[%s]", r.FrameBase.String(), "todo", r.Body)
+}
+
+func (r *RocketMQMessage) IsReq() bool {
+	return r.isReq
+}
+
+func (r *RocketMQStreamParser) ParseStream(streamBuffer *buffer.StreamBuffer, messageType protocol.MessageType) protocol.ParseResult {
+	buffer := streamBuffer.Head().Buffer()
+	if len(buffer) < 8 {
+		return protocol.ParseResult{
+			ParseState: protocol.NeedsMoreData,
+		}
+	}
+
+	frameSize := int(binary.BigEndian.Uint32(buffer[:4]))
+	if frameSize > len(buffer) {
+		return protocol.ParseResult{ParseState: protocol.NeedsMoreData}
+	}
+
+	headerLength := binary.BigEndian.Uint32(buffer[4:8])
+	headerDataLen := headerLength & 0xFFFFFF
+	serializedType := byte((headerLength >> 24) & 0xFF)
+
+	if len(buffer) < 8+int(headerDataLen) {
```

Comment:
Thr 4 and 8 are magical numbers here. I would like to suggest you to add constants